static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -81,6 +81,14 @@ define([
     };
 
     return [
+        new EditionTest('UK', 'MembershipEngagementBannerUkTest13', '2016-11-24', '2017-1-5', 'gdnwb_copts_mem_banner_aubanner__')
+            .addMembershipVariant('control', {})
+            .addMembershipVariant('3_rotating', {messageText: [
+                'We all want to make the world a fairer place. We believe journalism can help – but producing it is expensive. That’s why we need Supporters.',
+                'Become a Supporter and appreciate every article, knowing you’ve helped bring it to the page. Be part of the Guardian.',
+                'Not got round to supporting us yet? If everyone chipped in, our future would be more secure.'
```

Comment:
Seems that both 'around' & 'round' are acceptable?

http://ell.stackexchange.com/questions/51525/get-round-around-to-something-meaning-and-its-origin
https://www.collinsdictionary.com/dictionary/english/get-round